gitnamehere/point-of-sale-group-project
# Api authentication

This PR makes some api endpoints require authentication, since they are only supposed to be used by a user in the POS

Testing:
* `npm run start:dev`
* if you are logged into the POS, log out
* go to the store page
* verify everything works correctly
* try accessing `/api/orders`
* verify a 401 HTTP status is returned
* log in to the POS
* try accessing `/api/orders`
* verify it returns the orders as normal
